gammapy/data/observations.py

```diff
@@ -428,6 +431,37 @@ def read(cls, event_file, irf_file=None):
             **irf_dict,
         )
 
+    def write(self, path, overwrite=False, format='gadf-dl3', include_irfs=True):
+        """
+        Write this observation into `path` using the specified format
+        """
+        if format != 'gadf-dl3':
+            raise ValueError(f'Only the "gadf-dl3" format supported, got {format}')
+
+        path = make_path(path)
+
+        primary = fits.PrimaryHDU()
+        primary.header['CREATOR'] = f"Gammapy {__version__}"
+        primary.header['DATE'] = Time.now().iso
+
+        hdul = fits.HDUList([primary])
+
+        events = self.events
+        if events is not None:
+            hdul.append(events.to_table_hdu(format=format))
+
+        gti = self.gti
+        if gti is not None:
+            hdul.append(gti.to_table_hdu(format=format))
+
+        if include_irfs:
+            for irf_name in ['aeff', 'edisp', 'psf', 'bkg', 'rad_max']:
```

Comment:
To avoid the hard coded irf names you can use `Observation.available_irfs` here...

Reply:
That does not include `rad_max` at the moment. Is that just a place where we forgot to add it, or is there a specific reason?

Reply:
No, I think we just forgot it. It is the single places where the availability of irfs is checked without accessing the files on disk.

Reply:
ok, added `"rad_max"`